ai-machine-learning/thoth-thamos/advise.md

```diff
@@ -0,0 +1,49 @@
+Thamos CLI runs based on the `.thoth.yaml` config, which we have setup already in the example repository here. 
+
+You can browse the config file in the editor or via terminal using -
+
+``cat .thoth.yaml``{{execute}} 
+
+Now we are ready to do ask Thoth for Advise -
+
+``thamos advise``{{execute}}
+
+Tip - ``thamos advise --force`` runs the CLI without cache.
+
+Be a little patient here, this takes some time. After this is done you should see - 
+![thamos advise result](https://raw.githubusercontent.com/saisankargochhayat/katacoda-scenarios/master/thamos-cli/assets/thamos_advise_result.png)
+
+If you see the folder you would see a new file `requirement.txt` has been generated by Thoth.
+
+You can check both the files `requirements.in`, which has direct dependencies. 
+
+``cat requirements.in``{{execute}}
+
+And `requirements.txt`, which has dependencies along with digests - 
+
+``cat requirements.txt``{{execute}}
+
+You can check this out in the editor also. 
+
+Now lets try running the application after installing the depedendencies -
+
+``python3 -m venv venv``{{execute}}
+
+Then we activate this virtual environment -  
+
+``source venv/bin/activate``{{execute}}
+
+And install the depedencies then - 
+
+``pip3 install -r requirements.txt``{{execute}}
+
+And finally, run the application (the virtual environment needs to be still activated):
+
+``python3 ./app.py``{{execute}}
```

Comment:
When I tried to run this command in my katacoda env, I got this output:
![Screenshot from 2020-04-07 00-32-19](https://user-images.githubusercontent.com/22333506/78630549-04c25d00-7868-11ea-8762-0e0d06917801.png)

Reply:
I think that is the expected output.
@fridex

Reply:
Yes, that's expected output :+1: 

To remove warnings we can either adjust numpy or disable warnings in the code as discussed in https://github.com/tensorflow/tensorflow/issues/30427

To remove the instruction warning, we can possibly show it in the next step/scenario where thoth recommends optimized AICoE wheel files.

Reply:
as long it is intentional :+1: